src/nvim/os/env.c

```diff
@@ -464,12 +464,14 @@ char_u *vim_getenv(char_u *name, bool *mustfree)
 
       /* remove "doc/" from 'helpfile', if present */
       if (p == p_hf)
-        pend = remove_tail(p, pend, (char_u *)"doc");
+        pend = (char_u *) remove_tail((char *)p, (char *)pend, "doc");
 
       /* for $VIM, remove "runtime/" or "vim54/", if present */
       if (!vimruntime) {
-        pend = remove_tail(p, pend, (char_u *)RUNTIME_DIRNAME);
-        pend = remove_tail(p, pend, (char_u *)VIM_VERSION_NODOT);
+        pend = (char_u *) remove_tail((char *)p, (char *)pend,
+            RUNTIME_DIRNAME);
+        pend = (char_u *) remove_tail((char *)p, (char *)pend,
```

Comment:
It doesn't fit in a single line yet. But you can lineup argument on separated line below first argument on line before. Our style guide suggests that.